libpromises/evalfunction.c

```diff
@@ -238,8 +238,8 @@ static JsonElement* VarRefValueToJson(EvalContext *ctx, const FnCall *fp, const
             break;
 
         case RVAL_TYPE_CONTAINER:
-            convert = value;
-            *allocated = false;
+            convert = JsonCopy(value);
```

Comment:
The title of the commit "Avoid de-constifying assignment, fixes compiler warning." doesn't tell me much, unfortunately.